targets/targets.json

```diff
@@ -9850,6 +9850,25 @@
         },
         "forced_reset_timeout": 3
     },
+    "S5JS100": {
+        "inherits": ["Target"],
+        "core": "Cortex-M7",
+        "supported_toolchains": ["GCC_ARM", "IAR", "ARMC6"],
+        "default_toolchain": "GCC_ARM",
+        "extra_labels": ["Samsung", "SIDK_S5JS100"],
+        "macros": ["CMSDK_CM7", "MBED_MPU_CUSTOM",  "CMSIS_NVIC_VIRTUAL", "MBEDTLS_CONFIG_HW_SUPPORT"],
```

Comment:
Why do we use `"CMSIS_NVIC_VIRTUAL"` it seems that the table is at address 0 and it's a M7 so should have a VTOR.

Reply:
@bulislaw 
S111 target uses '0' address after OS bootup, every vector re-install at 0, is this patch acceptable?
https://github.com/andrewc-arm/mbed-os/commit/f0d1f2fbbbf027274012d673279184c7e1f97f0d

Reply:
Hi, @jh6186
On top of your VTOR setting to zero, I also removed NVIC virtual via e75292a11402e88a33dc25b6fbc03095604ae860. The test seems to be working well. If you have concerns, please reopen this review item.